### PR TITLE
fix limit 50 images duplicate

### PR DESCRIPTION
**Pull request by apt42541**
i trying to fix problem 50 images limit and duplicate on another page of webmota. it's my first time to pull a request.

### CODE REVIEW

File: src/web/mjs/connectors/Baozimh.mjs [hunk 1]
```diff
@@ -1,63 +1,76 @@
...
+        // let result = data2.substr(0, data2.lastIndexOf("/"));
+        const lastArray = pagesList2[pagesList2.length -1 ];
+        const ur2 = data2.substr(0, data2.lastIndexOf('/')+1);
+        const maxCount = data2.substr(data2.lastIndexOf('/') +1).split('.jpg')[0];
+        console.log(lastArray);
+        for (let i = 1; i < maxCount; i++) {
+            console.log(`${ur2}${i}.jpg`);
+            pagesList.push(`${ur2}${i}.jpg`);
+        }
+        return pagesList;
```
Comment (Druzai) [inline src/web/mjs/connectors/Baozimh.mjs]: To fix this issue just add filtering for duplicates in return.
```suggestion
        return pagesList.filter((page, index) => {
            return index === pagesList.findIndex(item => page === item);
        });
```

Comment (Druzai) [reply src/web/mjs/connectors/Baozimh.mjs]: Sorry that didn't answer you earlier in #5073, thought this was problem with the site.

**Comment by Druzai** [comment]
Also, I think you can remove your debugging code. You can test without this code. If the connector will still work correctly, then this issue is only about filtering for duplicates.

Remove: [lines 42 - 43](https://github.com/apt42541/hakuneko-webmota/blob/1f2dc4c769ad368278db08d8858fe676db490347/src/web/mjs/connectors/Baozimh.mjs#L42-L43), [lines 65 - 73](https://github.com/apt42541/hakuneko-webmota/blob/1f2dc4c769ad368278db08d8858fe676db490347/src/web/mjs/connectors/Baozimh.mjs#L65-L73)
And restore [lines 60 - 61](https://github.com/apt42541/hakuneko-webmota/blob/1f2dc4c769ad368278db08d8858fe676db490347/src/web/mjs/connectors/Baozimh.mjs#L60-L61) to:
```js
            pagesList.push( ...data.map(element => element.getAttribute('src')).filter(page => page) );
```

**Comment by 27136773** [comment]
Check and fix the `lint` errors.
Also make sure the file is saved in correct encoding (e.g. UTF-8), so only the important changes are shown in this PR (currently it shows [all 70 lines as changed](https://github.com/manga-download/hakuneko/pull/5088/files)).

**Comment by Sheepux** [comment]
bump
Rotting PR that only requires lint fix

**Comment by Druzai** [comment]
@Sheepux Can I create another PR to merge changes from this PR?
Cause I don't think original author will fix lint issue soon.

**Comment by 27136773** [comment]
> @Sheepux Can I create another PR to merge changes from this PR? Cause I don't think original author will fix lint issue soon.

Yes you can, its up to the maintainers to decide which PR gets in and why

**Pull request closed without merging**